src/globus_sdk/services/transfer/data.py

```diff
@@ -80,8 +81,9 @@ class TransferData(dict):
         task to fail.
         [default: ``False``]
     :type fail_on_quota_errors: bool, optional
-
-    Any additional parameters are fed into the dict being created verbatim.
+    :param additional_fields: additional fields to be added to the transfer
+        document. Mostly intended for internal use
+    :type additional_fields: dict, optional
```

Comment:
I wonder if maybe we should just remove this?
Maybe leave it for now, but I'd be okay with removing it.